### PR TITLE
Add GPIO Tutorial Series

### PR DESCRIPTION
**Pull request by Liquidscroll**
# Description

This pull request is a submission for adding a tutorial structure outline and a series of 4 new tutorials and their proposals covering General Purpose Input/Output (GPIO) functions to the GitHub repo. The tutorials cover the topics:
- Blinking an LED. [Complete.]
- Reading a button press. [Complete.]
- Using Pulse-Width Modulation (PWM) to control LED brightness. [Complete.]
- Using button presses and PWM to control LED brightness physically.

The purpose of this pull request is to contribute these proposal's and tutorials and to organise them into appropriate folders.

## Changes Made

- Added GPIO Tutorial Series Outline
- Added 4 new tutorial proposals:
	- 1.1 Blink an LED [Added Tutorial]
	- 1.2 Reading a button. [Added Tutorial]
	- 1.3 Use PWM to control LED brightness. [Added Tutorial]
	- 1.4 Modify PWM through button presses. 

Current images are examples as they are going to be redone to be much neater.

# How Has This Been Tested?

## Testing Checklist:

- [x]  Prettier formatting applied to all added content.
- [x]  Vale style guidelines checked and adhered to.
- [x] Tested in latest Chrome
- [x] Tested in latest Firefox

# Checklist

Please make sure you have completed the following before merging this pull request:

- [x]  My submission follows the style guidelines of this project.
- [x]  My submission is based on the relevant template in `Instructions and Templates/`.
- [x]  My tutorial is based on an approved tutorial proposal.
- [x]  I have requested a review from Aditya  on the Pull Request.

### CODE REVIEW

**Comment by Liquidscroll** [comment]
[Initial GPIO Tutorial Structure and Proposals](https://trello.com/c/Ai2l5Zft/476-initial-gpio-tutorial-structure-and-proposals)

File: Tutorial Proposals/GPIO Series/Digital IO/1.1 Blink an LED.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+This tutorial is designed for beginners who are interested in learning how to interact with the
+physical world using the Raspberry Pi. The reader learns how to wire an LED to the Raspberry Pi
+and control it to blink at regular intervals. This project is a foundational exercise in
+understanding General Purpose Input/Output (GPIO) pins.
+
+## Prerequisites
+
+- Basic understanding of electronics.
+- Raspberry Pi setup with SplashKit installed.
```
Comment (Wills2022) [inline Tutorial Proposals/GPIO Series/Digital IO/1.1 Blink an LED.md:15]: Might be useful to  put a link here with how to do this E.G.

Raspberry Pi setup with SplashKit [installed](https://splashkit.io/installation/linux/step-1/).

**Review by Wills2022 — APPROVED** [review]
Very thorough. Other than adding a  couple of links to other splashkit resources (I.E. the installer) there isn't much I can think to add here.

**Review by omckeon — APPROVED** [review]
Fantastic!

**Pull request merged**